client/components/happychat/button.jsx

```diff
@@ -21,6 +22,8 @@ export class HappychatButton extends Component {
 		allowMobileRedirect: PropTypes.bool,
 		borderless: PropTypes.bool,
 		primary: PropTypes.bool,
+		floating: PropTypes.bool,
+		offset: PropTypes.bool,
```

Comment:
Noting that by the name of the prop one could infer that it's a number value. I'd suggest renaming it to `hasOffset` or `withOffset`.

Reply:
I renamed the prop to `withOffset` and the CSS class to `with-offset`. It's like saying "make the button floating" or "make it floating with offset".

Reply:
Makes sense 👍  Thanks!